### ISSUE TITLE
After deploy I get a 404 main.xxxxxx.js

### ISSUE THREAD

**Issue by martinrf** [mars/create-react-app-buildpack#65]
After deploy I get a 404 forcing me to hard refresh, I added "headers": { "/**": { "Cache-Control": "no-store, no-cache" } } as suggested in Issue#52 but that not seems to work. 

Is there any other thing that I am missing on my static.json?

static.json:
{
  "root": "build/",
  "headers": { "/**": { "Cache-Control": "no-store, no-cache" } } 
}

**Comment by mars** [comment]
I just performed a fresh install of create-react-app and deployed a new app using the [Quick Start instructions](https://github.com/mars/create-react-app-buildpack#user-content-quick-start). Then, I visited the app in two browsers, Safari & Firefox. Finally, I made a small change to `App.js`, redeployed, and that change appears when I refresh the site.

So, because this buildpack + the default create-react-app code does not exhibit the issue, there must be something going on in your specific app to make this happen.

A few notes about how caching works with create-react-app:

* Everytime the app is rebuilt by `npm run build`, such as when deploying to Heroku, the cache-busting hash changes for the CSS & JS bundles, e.g. `/static/js/main.f0144275.js` is updated to `/static/js/main.67e36665.js`. The references in `index.html` are automatically updated to use these new assets. As a result, caching is enabled by default. Disabling it with no-cache headers will make performance suffer.
* Recent versions of create-react-app began including a Service Worker to improve low-connectivity & offline performance. This might be causing issues or at least confusing the debugging process. See the docs for [Making a Progressive Web App](https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md#user-content-making-a-progressive-web-app)

Debugging these issues can be frustrating! I'll do my best to help. Has your app always exhibited this problem or is did it just appear after a commit or and update? Try making a branch to reduce the problem. Delete pieces of your app until you see the problem disappear, then add back the code that manifests the problem. Remember that reducing the problem is the best way to communicate it to others and will frequently reveal the issue.
